tests/e2e/General/UsageTest.php

```diff
@@ -1090,6 +1092,24 @@ public function testCustomDomainsFunctionStats(array $data): void
 
         $this->assertEquals(200, $response['headers']['status-code']);
 
+        $rule = $this->client->call(
```

Comment:
Can we try to remove this addition, and see if tests pass?
If without this test fail, lets keep it. But I think we dont need it.

Reply:
Tests fail if we remove this